app/src/main/java/org/mifos/mobile/ui/client_accounts/AccountsScreen.kt

```diff
@@ -0,0 +1,933 @@
+                )
+            }
+
+            Row(modifier = Modifier.fillMaxWidth(), horizontalArrangement = Arrangement.SpaceBetween)
+            {
+                Row() {
+                    Text(
+                        text = "pending",
+                        style = MaterialTheme.typography.labelLarge,
+                        color = colorResource(id = R.color.gray_dark),
+                    )
+
+                    Text(
+                        text = " ${shareAccount.totalPendingForApprovalShares}",
+                        style = MaterialTheme.typography.bodyLarge,
+                        color = colorResource(id = R.color.black),
+                    )
+                }
+
+                if(setSharingAccountDetail){
+                    Row()
+                    {
+                        Text(
+                            text = "approved",
+                            style = MaterialTheme.typography.labelLarge,
+                            color = colorResource(id = R.color.gray_dark),
+                        )
+
+                        Text(
+                            modifier = Modifier.padding(end=12.dp),
+                            text = " ${shareAccount.totalApprovedShares}",
+                            style = MaterialTheme.typography.bodyLarge,
+                            color = colorResource(id = R.color.black),
+                        )
+
+                    }
+                }
+            }
+
+
+        }
+
+        Spacer( Modifier.weight(1f))
+
+
+    }
+}
+
+
+@Composable
+fun CustomShapeBox( color: Color) {
+    Box(
+        modifier = Modifier.background(Color.Transparent)
+    ) {
+        Canvas(modifier = Modifier
+            .height(60.dp)
+            .width(5.dp)) {
+            val radius = 10.dp.toPx()
+            val path = androidx.compose.ui.graphics.Path().apply {
+                moveTo(0f, 0f)
+                lineTo(size.width - radius, 0f)
+                arcTo(
+                    rect = androidx.compose.ui.geometry.Rect(
+                        offset = Offset(size.width - radius, 0f),
+                        size = Size(radius, radius)
+                    ),
+                    startAngleDegrees = -90f,
+                    sweepAngleDegrees = 90f,
+                    forceMoveTo = false
+                )
+                lineTo(size.width, size.height - radius)
+                arcTo(
+                    rect = androidx.compose.ui.geometry.Rect(
+                        offset = Offset(size.width - radius, size.height - radius),
+                        size = Size(radius, radius)
+                    ),
+                    startAngleDegrees = 0f,
+                    sweepAngleDegrees = 90f,
+                    forceMoveTo = false
+                )
+                lineTo(0f, size.height)
+                close()
+            }
+            drawPath(
+                path = path,
+                color = color
+            )
+        }
+    }
+}
```

Comment:
move to utils, and use proper naming, like not CustomShapeBox, something more specific